### PR TITLE
LocUpdPendIntent updated

### PR DESCRIPTION
**Pull request by fullmetaltac**
LocUpdPendIntent updated

### CODE REVIEW

File: android-o/AndroidPlayLocation/LocUpdPendIntent/LocUpdPendIntent/Properties/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.xamarin.LocUpdPendIntent">
-	<uses-sdk android:targetSdkVersion="26" />
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.xamarin.LocUpdPendIntent" android:installLocation="auto">
+	<uses-sdk android:targetSdkVersion="23" />
```
Comment (pjcollins) [inline android-o/AndroidPlayLocation/LocUpdPendIntent/LocUpdPendIntent/Properties/AndroidManifest.xml]: Both the manifest changes here seem unrelated, can we revert them?

File: android-o/AndroidPlayLocation/LocUpdPendIntent/LocUpdPendIntent/Utils.cs [hunk 2]
```diff
@@ -74,8 +77,22 @@ public static void SendNotification(Context context, string notificationDetails)
 	        // Get an instance of the Notification manager
 			var mNotificationManager = context.GetSystemService(Context.NotificationService) as NotificationManager;
 
-			// Issue the notification
-			mNotificationManager.Notify(0, builder.Build());
+		    // Android O requires a Notification Channel.
+		    if (Build.VERSION.SdkInt>= Build.VERSION_CODES.O)
+		    {
+		        var name = (Java.Lang.String)context.GetString(Resource.String.app_name);
```
Comment (pjcollins) [inline android-o/AndroidPlayLocation/LocUpdPendIntent/LocUpdPendIntent/Utils.cs]: This cast throws an exception on startup on Oreo devices. I'd suggest simply using
>  string name = context.GetString(Resource.String.app_name);

Comment (fullmetaltac) [reply android-o/AndroidPlayLocation/LocUpdPendIntent/LocUpdPendIntent/Utils.cs]: Changed.

File: android-o/AndroidPlayLocation/LocUpdPendIntent/LocUpdPendIntent/Properties/AndroidManifest.xml (hunk 1 above)
Comment (fullmetaltac) [reply android-o/AndroidPlayLocation/LocUpdPendIntent/LocUpdPendIntent/Properties/AndroidManifest.xml]: Reverted.
